content/2-5.md

```diff
@@ -1,42 +1,43 @@
+permalink: /curricula/developer-modules/forms/
+ref: /curricula/developer-modules/forms/
 lang: en
 github:
   repository: w3c/wai-curricula
   path: content/2-5.md
 license: creative-commons
+acknowledgements: /curricula/acknowledgements/
 footer: >
   <p><strong>Date:</strong> $Date</p>
-  <p><strong>Editors: </strong>Daniel Montalvo and <a href="http://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Contributors: <a href="https://www.w3.org/WAI/EO/EOWG-members">EOWG Participants</a>. </p>
+  <p><strong>Editors:</strong> Daniel Montalvo and <a href="http://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Contributors: <a href="https://www.w3.org/WAI/EO/EOWG-members">EOWG Participants</a>. ACKNOWLEDGEMENTS lists contributors and credits.</p>
   <p>Developed by the Education and Outreach Working Group (<a href="http://www.w3.org/WAI/EO/">EOWG</a>). Developed with support from the <a href="https://www.w3.org/WAI/about/projects/wai-guide/">WAI-Guide Project</a> funded by the European Commission (EC) under the Horizon 2020 program (Grant Agreement 822245).</p>
 navigation:
-  previous: /curricula/developing-accessible-content/tabular-information/
-  next: /curricula/developing-accessible-content/custom-widgets/
+  previous: /curricula/developer-modules/tables/
+  next: /curricula/developer-modules/custom-widgets/
 ---
 
 ## Introduction
 {:.no-display}
 
-Courses based on this module:
+Courses based on this module should:
 
-* explain use of proper code to label and group form controls, as well as to provide instructions and notifications for users to fill in forms
-* demonstrate how people with disabilities use form structures, labels, instructions, and notifications to understand and fill in form controls
+* demonstrate how people with disabilities use form structures, labels, instructions, and notifications to understand and interact with forms
+* explain coding techniques to mark up form controls, groups of form controls, and labels for form controls, as well as to provide instructions and notifications in accessible forms
 
 ## Learning Outcomes for Module
 
 Students should be able to:
 
-* label form controls so that they can be understood by users and interpreted by assistive technologies
-* group form controls that share a common purpose
-* mark up instructions that help understand form controls' purpose
-* notify users of time limits, errors, or successes during and after form completion
-* explain how labels, instructions, and notifications enable people with disabilities to navigate and fill in form controls
+* explain strategies that people with disabilities use to understand, navigate, and interact with forms
+* mark up form controls and labels for form controls with clear association between controls and labels
+* mark up groups of form controls that share a common purpose and provide labels for such groups
+* mark up instructions, notifications, and other feedback so that they are perceivable for all users
```

Comment:
"perceivable *by* all users" not "perceivable *for* all users"

Reply:
Will change.